ain-blockchain/staking/staking.md

```diff
@@ -0,0 +1,199 @@
+# Staking
+
+## What is staking?
+
+Staking is a way to lock your tokens to help the blockchain network or applications run smoothly. In return, you may earn rewards or contribute to the stability of the system.
+Think of it like putting your money in a savings account: you're helping the system grow, and sometimes you earn interest.
+
+## How does staking work?
+
+In our system, staking works a bit differently depending on where you stake your tokens:
+
+1. **Staking in the consensus app**
+
+- **What it does:** Your tokens are used to create new blocks on the blockchain.
```

Comment:
How about just mentioning that AIN Blockchain uses PoS consensus algorithm and the staking amount with the 'consensus' app is used for the purpose?

Reply:
done!